mdonatello/tests/test_mdonatello.py

```diff
@@ -4,16 +4,55 @@
 
 # Import package, test suite, and other packages as needed
 import mdonatello
+from mdonatello import MoleculeVisualizer
 import pytest
 import sys
+from ipywidgets import VBox, HTML, Dropdown, Checkbox, Button
+from rdkit import Chem
+from rdkit.Chem import AllChem
+from MDAnalysis import Universe
 
+# Dummy data for testing
+mol = Chem.MolFromSmiles("CCO")
+AllChem.Compute2DCoords(mol)
```

Comment:
Ah. I never understood the the difference between when it is like this and in pytests. Would the pytest not also create the Molecule, does it then create it once in the pytest fixture and here multiple times and with b you mean if it changes it can lead to errors in parallel tests?